Marlin/Configuration_adv.h

```diff
@@ -527,6 +527,13 @@ const unsigned int dropsegments = 5; //everything with less than this number of
     #define AUTO_FILAMENT_CHANGE                //Extrude filament until you press the button on LCD
     #define AUTO_FILAMENT_CHANGE_LENGTH 0.04    //Extrusion length on automatic extrusion loop
     #define AUTO_FILAMENT_CHANGE_FEEDRATE 300   //Extrusion feedrate (mm/min) on automatic extrusion loop
+ 
+  #if ENABLED(MANUAL_FILAMENT_CHANGE)
+    #define FILAMENT_LOAD_LENGTH 55 //i guess it's in cm
```

Comment:
Yes i know i've put this way to remenber to discuss with u.
Because in my last code i used mm, so my bowden is 55cm distante, i was using 550, but when i tested this one,